app/schema_validation/__init__.py

```diff
@@ -107,17 +107,13 @@ def validate(
         current_app.logger.info('Validation failed for: %s', json_to_validate)
         raise ValidationError(build_error_message(errors))
 
-    try:
-        if json_to_validate.get('personalisation'):
-            json_to_validate['personalisation'], errors = decode_personalisation_files(
-                json_to_validate.get('personalisation', {})
-            )
-            if len(errors) > 0:
-                error_message = json.dumps({'status_code': 400, 'errors': errors})
-                raise ValidationError(error_message)
-    except AttributeError:
-        current_app.logger.info('Validation failed for: %s', json_to_validate)
-        raise ValidationError('Payload is not json.')
+    if json_to_validate.get('personalisation'):
+        json_to_validate['personalisation'], errors = decode_personalisation_files(
+            json_to_validate.get('personalisation', {})
```

Comment:
You already know that the key is present with a value that is boolean True because you tested this on line 110.
```suggestion
            json_to_validate['personalisation']
```